src/app/pages/TransactionDetailPage/index.tsx

```diff
@@ -131,8 +145,13 @@ export const TransactionDetailView: FC<{
           </dd>
 
           <dt>{t('common.status')}</dt>
-          <dd>
+          <dd style={{ flexWrap: 'wrap', gap: '10px' }}>
             <TransactionStatusIcon success={transaction.success} withText={true} />
+            {transaction.error && (
+              <ErrorBox>
+                {transaction.error.message} (error code {transaction.error.code})
```

Comment:
let's move "error code" label to i18 next

Reply:
Done